starknet_py/tests/e2e/account/account_test.py

```diff
@@ -383,15 +374,22 @@ async def test_deploy_account_raises_on_incorrect_address(
 
 @pytest.mark.asyncio
 @pytest.mark.parametrize(
-    "call_contract",
+    "call_contract, corresponding_client",
```

Comment:
this is not important, but maybe just `client`? 😄

Reply:
Thought it would break the test since we have a fixture called `client`, but seems to be working!